src/AccessibilityInsights.Rules/Rules.cs

```diff
@@ -27,6 +33,12 @@ public static class Rules
             return dictionary;
         }
 
+        /// <summary>
+        /// Run a single rule based on the given RuleID.
+        /// </summary>
+        /// <param name="id"></param>
```

Comment:
>param [](start = 13, length = 5)

Should we have meaningful comments for id, element, and returns?

Reply:
I think additional comments would be both redundant and superfluous. But perhaps I'm not imagining how these parameters could be misinterpretted. Suggestions?

Reply:
id: The Id of the rule to run
element: The element to analyze
return: The output of the analysis

(As an aside, I'd probably name id "ruleId", but that's not a big deal)